chatgpt-shell.el

```diff
@@ -39,6 +39,7 @@
 (require 'eshell)
 (require 'find-func)
 (require 'ielm)
+(require 'pcsv)
```

Comment:
`pcsv` doesn't look to be a built-in package. We should move `(require 'pcsv)` inside `chatgpt-shell-load-awesome-prompts`, so existing users without this package can still load `chatgpt-shell`.

Reply:
Thanks, that's the bit I wasn't sure about where that dep should belong. I guess the requirement should be documented somewhere too?

Reply:
Thanks for the change. I'll follow up with a small addition (unless you're keen): anyone attempting to use the function without the dependency will be told to install it.